mayastor/src/pool.rs

```diff
@@ -131,3 +136,32 @@ impl From<Pool> for rpc::Pool {
         }
     }
 }
+
+/// PoolArgs is used to translate the input for the grpc
+/// Create/Import requests which comtains name, uuid & disks.
+/// This help us avoid importing grpc structs in the actual lvs mod
+#[derive(Clone, Debug)]
+pub struct PoolArgs {
+    pub name: String,
```

Comment:
Should we define a PoolName type? Eg: are there constraints to how short/long the name can be?

Reply:
For now spdk does not have limit I guess but we can do this future